mkdocs_include_markdown_plugin/event.py

```diff
@@ -54,10 +54,16 @@ def found_include_tag(match):
 
         return text_to_include
 
+    def interpret_escapes(str):
```

Comment:
Please, don't overwrite locally the [builitin `str` type](https://docs.python.org/3/library/stdtypes.html#str) as parameter argument, use `value` instead.

Reply:
Wow, had no idea python used the identifier "str" as a builtin class name. Thanks for the heads up. May I use "strg" to indicate this is intended to run on strings?

Reply:
You can use Mypy type checking like in [this function](https://github.com/mondeja/mkdocs-include-markdown-plugin/blob/83cd6601e105822705a7b8740d03b92e222a919b/mkdocs_include_markdown_plugin/process.py#L92) to specify clearly that is intended to run on strings.